optimize/self-managed/optimize-deployment/configuration/setup-event-based-processes.md

```diff
@@ -33,12 +33,12 @@ A full configuration example authorizing the user `demo` and all members of the
 ## Use Camunda activity event sources for event based processes
 
 :::note Authorization to event-based processes
-When Camunda activity events are used in event-based processes, Camunda Admin Authorizations are not inherited for the event-based process. The authorization to use an event-based process is solely managed via the access management of event-based processes when [publishing an event-based process]($optimize$/components/userguide/additional-features/event-based-processes/#publishing-an-event-based-process) or at any time via the [Edit Access Option]($optimize$/components/userguide/additional-features/event-based-processes/#event-based-process-list---edit-access) in the event-based process List.
+When Camunda activity events are used in event-based processes, Camunda Admin Authorizations are not inherited for the event-based process. The authorization to use an event-based process is solely managed via the access management of event-based processes when [publishing an event-based process](components/userguide/additional-features/event-based-processes.md/#publishing-an-event-based-process) or at any time via the [Edit Access Option](components/userguide/additional-features/event-based-processes.md/#event-based-process-list---edit-access) in the event-based process List.
```

Comment:
The files were largely moved verbatim from `docs/` to `optimize/`, but links were also updated -- these _used_ to be cross instance, but now that everything is in the `optimize/` folder they can go back to being plain-old-.md-references. 🙌